m4_sessions/1803-inning/group_7.md

```diff
@@ -1,13 +1,48 @@
 Group 7
 
-Group Members:
+Group Members: 
+Tyler Madsen, Lauren Billington, Gabe Afflitto, Max Stackhouse
 
-Name of Session:
+Name of Session: 
+Finding Meaningful Common Ground
 
 Session Description (Why should students come to this session):
+Strategies to finding connections with new people to build closer relationships with others.
 
 Session Outcomes (What will students learn/leave with?):
+Phrases / strategies that they can use.
+Unseen common ground
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+1:15
+10m - Short Introduction (why this is important)
+5m (15) - Explain Bingo
+10m (25) - Bingo card of things you've done, find your matches
+5m (30) - Explain Observation Connection
+10m (40) - Observation Connection (someone you know or don't know)
+5m (45) - Explain Finding New Common Ground
+10m (55) - Find someone you know and find new common ground
+5m (60) - Cultivating Common Ground
```

Comment:
Would love to have a clear take away for students:  What actions will they take now?  What will they apply based on this session?  Can they use this to find a mentor?  Find connections with current students?